### PR TITLE
Preconnect payments

### PR DESCRIPTION
**Pull request by dvoytenko**

### CODE REVIEW

File: src/runtime/pay-flow.js [hunk 1]
```diff
@@ -33,6 +33,13 @@ const PAY_REQUEST_ID = 'swg-pay';
  */
 export class PayStartFlow {
 
+  /**
+   * @param {!../utils/preconnect.Preconnect} pre
+   */
+  static preconnect(pre) {
+    pre.prefetch(feUrl('/pay'));
```
Comment (cramforce) [inline src/runtime/pay-flow.js]: Please make sure this actually results in cache hits. Is this for an iframe? I've seen this not work before (at least in terms of not coalescing request if it hasn't come back by iframe usage.`prefetch` is semantically for future navigations.

I'd recommend to also preconnect

Comment (dvoytenko) [reply src/runtime/pay-flow.js]: This is for a plain navigation and I definitely see cache hits everywhere, except Safari. But Safari doesn't do the fetch anyway - so not a net-negative.

Do you know if I can do `<link rel="preconnect prefetch">`?

Comment (cramforce) [reply src/runtime/pay-flow.js]: Yes you can! Sounds good for plain navigation!!!

Comment (dvoytenko) [reply src/runtime/pay-flow.js]: Thanks! Updated.

**Pull request merged**